src/frontend/js/components/DashBoard/Navigator/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { NavLink, useRoutes } from 'react-router-dom';
```

Comment:
In order to keep folder structure as flat as possible. I think we could rename this file `Navigator.tsx` then move it at the root of `js/components/Dashboard`.